extensions/wikia/MercuryApi/models/MercuryApi.class.php

```diff
@@ -125,6 +127,9 @@ public function getWikiVariables() {
 			'mainPageTitle' => Title::newMainPage()->getPrefixedDBkey(),
 			'theme' => SassUtil::getOasisSettings(),
 			'wikiCategories' => WikiFactoryHub::getInstance()->getWikiCategoryNames( $wgCityId ),
+			'analytics' => [
+				'UAEnabled' => $wgAnalyticsProviderUseUA,
+			],
```

Comment:
`$wgAnalyticsProviderUseUA` does not have a default value defined. Please use `!empty($wgAnalyticsProviderUseUA)` instead.